cmd/export.go

```diff
@@ -19,12 +19,14 @@ var channels []string
 var channelWithChildren []string
 var outputDir string
 var metadataOnly bool
+var startingDate string
 
 func init() {
 	exportCmd.Flags().StringSliceVar(&channels, "channels", nil, "Channels to be exported")
 	exportCmd.Flags().StringSliceVar(&channelWithChildren, "channel-with-children", nil, "Channels to be exported")
 	exportCmd.Flags().StringVar(&outputDir, "outputDir", ".", "Location for generated data")
 	exportCmd.Flags().BoolVar(&metadataOnly, "metadataOnly", false, "export only metadata")
+	exportCmd.Flags().StringVar(&outputDir, "startingDate", "", "The date of the last synchronization")
```

Comment:
@rjmateus What should be the fallback value for the date? Especially in the case of a fresh new sync. Not it's set to an empty string which evaluates to an error when validated.

Reply:
I would say to nil, and if the value is nil we shouldn't apply the filter. I think we should also validate the date-time formate, and put the format on the info message.

Reply:
Well, golang has no such thing as a `nil` string - the zero value for type string is indeed `""`

https://go.dev/tour/basics/12

FMPOV it's OK leaving the empty string as the default. Alternatively we can use an obviously very long past value such as `1970-1-1`, but, from a user perspective, I fear that will be more confusing.

I also suggest the following:

```suggestion
	exportCmd.Flags().StringVar(&startingDate, "packagesOnlyAfter", "", "Only export packages added or modified after the specified date")
```

Note that the wrong variable was passed (`&outputDir`) and the fact we are specific, at least for now, this exclusively applies to software channels.